spec/controllers/students_controller_spec.rb

```diff
@@ -103,8 +103,7 @@
   end
 
   describe 'User is a Student' do
-    include ERB::Util
```

Comment:
This is good---there's going to be a merge conflict here because I merged in https://github.com/MarkUsProject/Markus/pull/4922/files#diff-5f44c1c580dbc94788e40f1aecf6f711215f30ce917214dc8d25765105cc4a37R15. (FYI)

Reply:
nice! but it looks like it handled the merge ok